corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -438,40 +527,62 @@ def _make_report_filter(conf):
             }
 
         filter_configs = self.cleaned_data['filters']
-        return [_make_report_filter(f) for f in filter_configs]
+        filters = [_make_report_filter(f) for f in filter_configs]
+        if self.source_type == 'case':
+            self._convert_closed_filter_to_choice_list(filters)
+        return filters
+
+    @classmethod
+    def _convert_closed_filter_to_choice_list(cls, filters):
+        for f in filters:
+            if f['field'] == get_column_name('closed') and f['type'] == 'dynamic_choice_list':
+                f['type'] = 'choice_list'
+                f['choices'] = [
+                    {'value': 'True'},
+                    {'value': 'False'}
+                ]
 
     @property
     def _report_charts(self):
         return []
 
 
-class ConfigureNewBarChartReport(ConfigureNewReportBase):
-    group_by = forms.ChoiceField()
-    form_title = "Configure Bar Chart Report"
+class ConfigureBarChartReportForm(ConfigureNewReportBase):
+    group_by = forms.ChoiceField(label="Property")
+    report_type = 'chart'
 
-    def __init__(self, app_id, source_type, report_source_id, *args, **kwargs):
-        super(ConfigureNewBarChartReport, self).__init__(app_id, source_type, report_source_id, *args, **kwargs)
+    def __init__(self, report_name, app_id, source_type, report_source_id, existing_report=None, *args, **kwargs):
+        super(ConfigureBarChartReportForm, self).__init__(report_name, app_id, source_type, report_source_id, existing_report, *args, **kwargs)
         self.fields['group_by'].choices = self._group_by_choices
 
+        # Set initial value of group_by
+        if self.existing_report:
+            existing_agg_cols = existing_report.aggregation_columns
+            assert len(existing_agg_cols) < 2
```

Comment:
It's possible that this report was edited outside of the report builder (although this isn't intended to ever happen). The UI only supports a single aggregation column.